### ISSUE TITLE
Contrast themes – Tippy tooltip arrows render as rectangles in WHCM

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#8835]
Related: #8830. Our Tippy tooltips are also:

- Missing a border or outline, so it’s hard to tell where the dropdown is
- Showing their arrows as rectangles

![Image](https://user-images.githubusercontent.com/877585/178253989-70e106b1-3aca-4733-bcb0-6d3c749c20a5.png)

Figma link: https://www.figma.com/file/O81E48wieOZ7e6AivvksWn/Contrast-themes%3A-Wagtail-audit?node-id=6805%3A4110

## Border / outline

We should add a relatively thick border / outline so the dropdown is easy to identify. We could try a few different styles to figure out what feels right.

## Tooltip arrow

Discussing this on the [web-a11y Slack](https://web-a11y.slack.com/archives/C02D66WBDTQ/p1657203369545429), a few options were proposed:

- Use a unicode arrow (`content: '\25B2';`)
- Use a clip path (`clip-path: polygon(50% 0, 0 100%, 100% 100%).`)
- Use a background-image (but I would have expected this to not show in WHCM? would require `forced-color-adjust: none` as well?)
- Use `forced-color-adjust: none;` (but we then lose the correct color from WHCM)

The best option seems to me like clip-path, or otherwise the unicode arrow. It would be worth trying a few just to be sure.

**Comment by Julietadeboye** [comment]
I would like to work on this

**Comment by Julietadeboye** [comment]
Recreating this

**Comment by xomcodes** [comment]
I would work on this

**Comment by lb-** [comment]
@xomcodes - let's give Juliet a chance to recreate this. 

@janehughes - if it's ok we might see if someone looking at applying to Outreachy can pick this up.
